packages/ui/src/global.css

```diff
@@ -15,7 +15,7 @@
 }
 
 @layer components {
-    .cui-switch {
+    button.cui-switch {
```

Comment:
why this css class need to be tied to button?

Reply:
Otherwise Tailwind's base CSS overrides the switch style.